README.md

```diff
@@ -56,17 +56,12 @@ npm install @google-cloud/profiler
 
 ### Prerequisites
 
-1. Your application will need to be using Node.js 10.4.1 or greater on the 10.x
- version branch or Node.js 12.15.0 or lower on the 12.x version branch. The 
- profiler will not be enabled when using version prior to 10.4.1 or after 
- 12.15.0. This is because:
- * Versions of Node.js 10 prior to 10.4.1 are impacted by
-   [this](https://bugs.chromium.org/p/chromium/issues/detail?id=847863) issue,
-   which can cause garbage collection to take several minutes when heap 
-   profiling is enabled.
- * Versions of Node.js 12 after 12.15.0 are impacted by 
```

Comment:
This will need to be updated one there's a fix for the memory leak, but Profiler will not ever be able to support currently released versions of Node 12 that are after 12.15, or currently released versions of Node 14. The memory leak is a part of V8 code, so new releases of Node.js are required for profiler to not cause a memory leak.

Reply:
Marking this as resolved. We have workaround for memory leak; so I needed to revert this.